src/components/manage/BlockChooser/BlockChooser.jsx

```diff
@@ -143,10 +144,15 @@ const BlockChooser = ({
 
 BlockChooser.propTypes = {
   currentBlock: PropTypes.string.isRequired,
-  onMutateBlock: PropTypes.func.isRequired,
-  onInsertBlock: PropTypes.func.isRequired,
+  onMutateBlock: PropTypes.func,
+  onInsertBlock: PropTypes.func,
   allowedBlocks: PropTypes.arrayOf(PropTypes.string),
   blocksConfig: PropTypes.objectOf(PropTypes.any),
 };
 
-export default injectIntl(BlockChooser);
+export default injectIntl(
+  React.forwardRef((props, ref) => (
+    <BlockChooser {...props} blockChooserRef={ref} />
+  )),
+  { forwardRef: true },
```

Comment:
I would use `useIntl` because of this is a bit shady and not sure that it's supported in latest react-intl versions.

Reply:
Much better indeed. I've extracted this code from another, older PR that predates onInsertBlock... I should have given it a fresh look.